dbms/src/Functions/FunctionsGeo.cpp

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
```

Comment:
I think it is better to make this suppression only apply to `GeoUtils.h` line 32~44. We wrap those suppressions with "diagnostic push/pop" to keep the TiFlash code clean for those warnings.

https://github.com/pingcap/tiflash/blob/c6740e5c8d184d854447cd7d8e04939efe99de75/dbms/src/Functions/GeoUtils.h#L32-L44